core/build.gradle.kts

```diff
@@ -1,8 +1,13 @@
+import org.gradle.jvm.tasks.Jar
+
 plugins {
     kotlin("jvm") version "1.3.70"
+    id("org.jetbrains.dokka") version "0.10.0"
+    `maven-publish`
+    signing
 }
 
-group = "org.example"
+group = "io.github.AlexanderPrendota"
```

Comment:
Hm, maybe `com.kotlin.samples.verifier`

Reply:
Shouldn't it match the one from sonatype ticket?

Reply:
No idea. But let's change. I've just realized what it means ;) Sorry.
Is it possible?

Reply:
If I change group only in the gradle file, it fails to build.
I don't think I can make a new ticket with com.kotlin.samples.verifier as a GroupId since I'm not the owner or maintainer of this domain name 🤔

Reply:
Let's try